sysctl.d/50-pid-max.conf

```diff
@@ -11,6 +11,6 @@
 # (e.g. /etc/sysctl.d/90-override.conf), and put any assignments
```

Comment:
hmm, it depends on nr_cpus? where did you find that?

Reply:
in the [kernel](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/kernel/pid.c#n654). It's the _default max_, not the _possible max_.